README.rst

```diff
@@ -225,6 +225,12 @@ response will contain an error object with a ``debug_url``:
 
 This tells you to find the traceback interpreter at ``<host>:<port>/debug/1234567890``.
 
+Logging
+-------
+
+The registry has a default logger, ``Registry._logger`` and it logs all errors that are not defined
+by ``typedjsonrpc``. You can configure the logger in ``Registry._get_default_logger()``.
```

Comment:
if someone wants to change the logging settings, then they need to change the source code of registry.py... is that what @mnazbro meant in the issue by "configurable"?